ryan-deblois.txt

```diff
@@ -0,0 +1,45 @@
+
+1. What is Semantic HTML? Semantic HTML is the use of markup so that websites can read and present what is on the page correctly with meaning.
+    2. What is HTML used for? HTML is used to have a universal language for other to read and understand as well as computers to properly display what is
+	being asked. 
+    3. What is an attribute and where do we put it? An attribute is a value you give something in HTML such as color, font, alignment, etc.
+    4. What is the h1 tag used for? How many times should I use it on a page? h1 tag is used for the main heading on a page. It should only be used once.
+    5. Name two tags that have required attributes 
+    6. What do we put in the head of our HTML document? <header>
+    7. What is an id? a more specific tag for an element
+    8. What elements can I add an id to? Any element
+    9. How many times can I use the same id on a page? Only once.
+    10. What is a class? Another way to style an element, but can be multiple to the same class.
+    11. What elements can I add a class to? Any element.
+    12. How many times can I use the same class on a page? As many as needed.
+    13. How do I get my link to open in a new tab? Right click choose open in a new tab?
```

Comment:
How do you do this using HTML?

Reply:
You add a target="_blank" attribute to the link